File: veloProweb-frontend/src/app/models/DTO/product-dto.ts
import { StatusProduct } from "../enum/status-product";

export interface ProductDTO {
    id: number;
    description: string;
    salePrice: number;
    buyPrice: number;
    stock: number;
    status: boolean;
    statusProduct: StatusProduct;
    brand: string;
    unit: string;
    subcategoryProduct: string;
    category: string;
}

File: veloProWeb/src/main/java/com/veloProWeb/Model/DTO/ProductDTO.java
package com.veloProWeb.Model.DTO;

import com.veloProWeb.Model.Enum.StatusProduct;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class ProductDTO {
    private Long id;
    private String description;
    private int salePrice;
    private int buyPrice;
    private int stock;
    private boolean status;
    private StatusProduct statusProduct;
    private String brand;
    private String unit;
    private String subcategoryProduct;
    private String category;
}
